packages/client/src/features/export/export-modules.ts

```diff
@@ -46,5 +46,5 @@ export const standaloneExportModule = new FeatureModule(
         bind(ExportSvgActionHandler).toSelf().inSingletonScope();
         configureActionHandler(context, ExportSvgAction.KIND, ExportSvgActionHandler);
     },
-    { requires: exportModule }
+    { featureId: Symbol('standaloneExport'), requires: exportModule }
```

Comment:
The `exportModule` a few lines up could also profit from an actual Symbol because currently it just gets a number assigned.